traitsui/editor.py

```diff
@@ -710,7 +709,7 @@ def _str(self, value):
             or 3.
         """
         # In Unicode!
-        return six.text_type(value)
+        return str(value)
```

Comment:
Does the `_str` method ever get called? Can it be removed entirely? (I couldn't find any calls within TraitsUI, and in theory it's private, so no-one *outside* TraitsUI should be calling it.)

Reply:
Yes, looks like it can go.